src/quo2/components/tags/context_tag/style.cljs

```diff
@@ -37,21 +17,88 @@
 
 (def audio-tag-icon-color colors/white)
 
-(defn audio-tag-text-color
-  [override-theme]
-  (colors/theme-colors colors/neutral-100 colors/white override-theme))
-
-(def community-tag
-  {:padding-vertical 2
-   :padding-left     2})
+(defn container
+  [theme type size state blur? customization-color]
+  (let [background-color (if blur?
+                           (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5 theme)
+                           (colors/theme-colors colors/neutral-10 colors/neutral-90 theme))
+        border-radius    (cond
+                           (not (#{:account :collectible} type)) 16
+                           (= size 24)                           8
+                           :else                                 10)
+        border-color     (colors/custom-color-by-theme customization-color 50 60 nil nil theme)]
```

Comment:
imo this function is too messy with `nil`, `nil` params. 
I think you should use 
```
(colors/theme-colors 
(colors/custom-color customization-color 50)
(colors/custom-color customization-color 60)
theme)
```
and eventually we should just make the `50` and `60` based on the theme like we discussed in Porto - and use the overlay for customisation pressables. 
This will make this function much simpler. Also the opacity set in 99% of cases is the same so it's unnecessary for dark and light opacity imo.

Anyway, beyond this pr but we should address this at some point soon 👍

Reply:
Nice suggestion.

And yes! we should address it soon, since the code is growing more and more